docs/sources/panels/transformations/prereq.md

```diff
@@ -0,0 +1,23 @@
++++
+title = "Prerequisites"
+type = "docs"
+[menu.docs]
+identifier = "transformations_prerequisites"
+parent = "transformations"
+weight = 300
++++
+
+# Prerequisites
+
+Before you can configure and apply transformations:
+
+- You must have entered a query and returned data from a data source. For more information on queries, refer to [Queries]({{< relref "../queries.md" >}}).
```

Comment:
I suggest folding this topic into the Overview topic.

Reply:
Done.